### ISSUE TITLE
Helios KWL 360 W (EasyControls 3)

### ISSUE THREAD

**Issue by mgrfilipmarek** [asev/homeassistant-helios#18]
Can someone confirm that this component works well with Helios KWL 360 W?

**Comment by janesp** [comment]
I don't think so - it looks like EC3 is a totally different world (https://www.easycontrols.net/de/). My understanding - EC2 uses proprietary MODBUS protocols over IP (Ethernet cable), EC3 most likely REST API (probably WLAN).

I have a KWL EC 300 W (still struggling to connect to it). According to Helios, legacy models cannot be upgraded to EC3. I am currently following up if an electronic board swap would be possible.

**Comment by Cheese1300** [comment]
Hi janesp, do you have a solution for the KWL EC 300? Also trying to connect...
